src/Rating.soy

```diff
@@ -3,10 +3,46 @@
 /**
  * This renders the main element.
  * @param? elementClasses
+ * @param? label
  * @param id
- * @skipUpdates elementClasses
  */
 {template .render}
 	<div id="{$id}" class="rating component{$elementClasses ? ' ' + $elementClasses : ''}">
+		{if $label}
+			{call .rateLabel data="all" /}
+		{/if}
+		{call .rateItems data="all" /}
+		{call .rateInput data="all" /}
 	</div>
 {/template}
+
+/**
+ * @param? label
+ * @param id
+ */
+{template .rateLabel}
+	<span id="{$id}-rateLabel" class="rate-label">{$label}</span>
```

Comment:
Also, you don't need id for each template, not even on the main `render` template anymore. In fact, the `id` attribute doens't even exist anymore. :D
Just update to the new version and you can remove these.